pkg/operator/starter.go

```diff
@@ -227,11 +227,13 @@ var RevisionConfigMaps = []revision.RevisionResource{
 var RevisionSecrets = []revision.RevisionResource{
 	// these need to removed, but if we remove them now, the cluster will die because we don't reload them yet
 	{Name: "etcd-client"},
-	// this is needed so that the cert syncer itself can request certs.  It uses localhost
-	{Name: "kube-apiserver-cert-syncer-client-cert-key"},
 	{Name: "kubelet-client"},
 	// etcd encryption
 	{Name: "encryption-config", Optional: true},
+
+	// this needs to be revisioned as certsyncer's kubeconfig isn't wired to be live reloaded, nor will be autorecovery
+	{Name: "localhost-recovery-serving-certkey"},
```

Comment:
This is named like a serving cert.  If it's used for serving by the kube-apiserver, it should be live reloaded.

Reply:
it is also used by the clients that don't support live reloading (cert syncer and regeneration controller)

Reply:
is it only used by the sidecars such that live-reload is not needed?

Reply:
I'd phrase it so that the side cars can't handle live-reload so even the apiserver can't use live-reload even if it would support it as they share the file on disk.

Reply:
There is no other actor who will fall over a missing reload, right?

Reply:
the won't be missing reload, it will trigger new rollout

Reply:
ok